lib/public/views/Runs/Overview/RunsOverviewPage.js

```diff
@@ -37,21 +37,11 @@ export const RunsOverviewPage = ({ runs: { overviewModel: runsOverviewModel }, m
     return h('', [
         h('.flex-row.header-container.pv2', [
             filtersPanelPopover(runsOverviewModel, runsActiveColumns),
-            exportRunsButton({ runsOverviewModel, modalModel }),
+            exportRunsButton(runsOverviewModel, modalModel),
```

Comment:
For me, it is absolutely not obvious that the RunsButton will contain the whole popover.
`exportRunsTriggerAndModal` may be more explicit? Up to you.

Reply:
renamed